build/Targets/Versions.props

```diff
@@ -41,6 +41,10 @@
     <!-- NuGet version -->
     <NuGetReleaseVersion>$(RoslynFileVersionBase)</NuGetReleaseVersion>
     <NuGetPerBuildPreReleaseVersion>$(NuGetPreReleaseVersion)-$(BuildNumberFiveDigitDateStamp)-$(BuildNumberBuildOfTheDayPadded)</NuGetPerBuildPreReleaseVersion>
+    <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == ''">$(NUGET_PACKAGES)</NuGetPackageRoot> <!-- Respect environment variable if set -->
```

Comment:
Perhaps move nuget related properties to a separate NuGet.props like so:
https://github.com/dotnet/interactive-window/blob/master/build/Toolset/NuGet.props
https://github.com/dotnet/symreader-portable/blob/master/build/Toolset/NuGet.props
https://github.com/dotnet/symreader/blob/master/build/Toolset/NuGet.props
https://github.com/dotnet/symstore/blob/master/build/Toolset/NuGet.props